frontend/src/utils/acled-utils.ts

```diff
@@ -1,18 +1,20 @@
 import { sortBy } from 'lodash';
 import GeoJSON from 'geojson';
 import moment from 'moment';
-import { ACLED_ISO } from '../config';
 import { PointLayerData } from '../config/types';
 import { DEFAULT_DATE_FORMAT } from './name-utils';
+import { appConfig } from '../config';
 
 export const fetchACLEDDates = async (url: string): Promise<number[]> => {
-  if (!ACLED_ISO) {
+  const acledIso = appConfig.acled_iso;
+
+  if (!acledIso) {
     throw new Error(
-      'ACLED processing is defined in layers.json but environment variables were not found',
+      'ACLED processing is defined in layers.json. However, country iso code not found',
```

Comment:
Let's change this to:

However, no country ISO code was set in the prism.json config. Please set the "acled_iso" parameter.

Reply:
Included in 900bc75d13106e3c7aaeaae0cfabf047eb7d5992